addon/content/reducer.js

```diff
@@ -658,6 +657,25 @@ function messages(state = initialMessages, action) {
   }
 }
 
+const summaryActions = {
+  showMessagesInvolving({ name, email }) {
+    return async () => {
+      await browser.convContacts
+        .showMessagesInvolving({
+          name,
+          email,
+        })
+        .catch(console.error);
+    };
+  },
+  sendEmail({ name, email }) {
+    return async () => {
+      const dest = await browser.convContacts.makeMimeAddress({ name, email });
+      await browser.compose.beginNew({ to: [dest] }).catch(console.error);
```

Comment:
Unfortunately this won't quite work right for us. We still need the information about the folder and the associated identity - otherwise if you're on a different account you'll get the wrong from address when you select select.

Whilst we can add the [identity information with TB 76 and later](https://thunderbird-webextensions.readthedocs.io/en/latest/compose.html#composedetails), we can't for TB 68.

Therefore I think we need to reimplement the current code as an API for now.